kf_lib_data_ingest/cli.py

```diff
@@ -74,10 +72,13 @@ def ingest(dataset_ingest_config_path, target_url, use_async, log_level_name):
         root_dir, 'target_apis', 'kids_first.py')
 
     # Run ingest
-    DataIngestPipeline(
+    err = DataIngestPipeline(
         dataset_ingest_config_path, target_api_config_path, **kwargs
     ).run()
 
+    if err:
```

Comment:
Oh now I see why you return not passed in pipeline.run()... Hmm can we change it to, `no_errors` or `success` and then say if not `<success flag>` then exit 1?

Reply:
√